packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1038,7 +1039,7 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
             children: children,
             delegate: new _ScaffoldLayout(
               statusBarHeight: padding.top,
-              bottomPadding: widget.resizeToAvoidBottomPadding ? padding.bottom : 0.0,
+              bottomPadding: widget.resizeToAvoidBottomPadding ? math.max(padding.bottom, viewInsets.bottom) : 0.0,
```

Comment:
do you want this to resize to avoid padding.bottom? That doesn't seem right.

Reply:
I do very temporarily until an engine change lands. This will be viewInsets-only shortly. This should have included a TODO.

Reply:
Added the TODO in #13434.

Reply:
And gone in #13437